README.md

```diff
@@ -38,6 +38,24 @@ fn main() {
 
 ```
 
+To perform a ping using a domain name instead of an IP address, you can use any 3d-party DNS resolver or [`ToSocketAddrs`](https://doc.rust-lang.org/std/net/trait.ToSocketAddrs.html) from the standard library:
```

Comment:
```suggestion
To perform a ping using a domain name instead of an IP address, you can use any 3rd-party DNS resolver or [`ToSocketAddrs`](https://doc.rust-lang.org/std/net/trait.ToSocketAddrs.html) from the standard library:
```